src/components/ModalPortal.js

```diff
@@ -379,6 +379,7 @@ export default class ModalPortal extends Component {
       onClick: this.handleContentOnClick,
       role: this.props.role,
       "aria-label": this.props.contentLabel,
+      "aria-modal": true,
```

Comment:
You can set this property using `aria={{ "modal": true }}`. Not sure if it would be ok to set by default.

Any thoughts?

Reply:
Yes, I saw this, but it make sense to be here, it's not really a addition, but a basic for a good accessible modal. 

But of course, if the PR is rejected, I have a other solution with the aria additionnal properties, but I have the feeling that is place is by default.

Reply:
That's what I'm worried about...if this aria attribute, added by default, would conflict with any other accessibility role.

You can find more details here (W3C Modal)[https://www.w3.org/TR/wai-aria-practices/#dialog_modal].

Reply:
I based my proposal on the W3C recommandation, on the same link you can find a few lines below : 

https://www.w3.org/TR/wai-aria-practices/#dialog_roles_states_props

```
The dialog container element has aria-modal set to true.
```

But as you wish :)

Reply:
If you reject the PR please close it, that I'll have to make another PR in my Design System to correct it by my own

Reply:
You don't need to be a pessimist. :stuck_out_tongue_closed_eyes: 

Maybe if you move this to the line below the patch, people can control this attribute (not sure why someone would do that, but better to keep it editable).

What do you think?